app/controllers/snippets/snippets_controller.rb

```diff
@@ -1,45 +1,29 @@
-module Snippets
-  class SnippetsController < ApplicationController
+class Snippets::SnippetsController < JSONAPI::ResourceController
```

Comment:
I prefer the explicit module naming convention here - so using the verbose syntax rather than `Snippets::SnippetsController`.  I think it's clearer and there's no benefit from concatenation.